cohort-evaluator/src/main/java/com/ibm/cohort/cql/evaluation/CqlEvaluationRequest.java

```diff
@@ -85,32 +109,4 @@ public String toString() {
                 .append("contextValue", contextValue)
                 .toString();
     }
-
-	@Override
-	public boolean equals(Object o) {
-		if (this == o) return true;
-
-		if (o == null || getClass() != o.getClass()) return false;
-
-		CqlEvaluationRequest that = (CqlEvaluationRequest) o;
-
-		return new EqualsBuilder()
-				.append(descriptor, that.descriptor)
-				.append(expressions, that.expressions)
-				.append(parameters, that.parameters)
-				.append(contextKey, that.contextKey)
-				.append(contextValue, that.contextValue)
-				.isEquals();
-	}
-
-	@Override
-	public int hashCode() {
-		return new HashCodeBuilder(17, 37)
-				.append(descriptor)
-				.append(expressions)
-				.append(parameters)
-				.append(contextKey)
-				.append(contextValue)
-				.toHashCode();
-	}
```

Comment:
Pedantic: I don't see us actually using `CqlEvaluationRequest` within a `Set` or as a `Map` key, but is there any reason to not keep `hashCode()` and `equals()` here?